source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,577 +3,765 @@
+
+.. code-block:: console
+
+   "Bundles": ["os-core", "os-core-update", "kernel-native"]
+
+Next, set the `Version` field to the mix version content mixer should use to
+build image. `ister` allows you to build an image from any mix version you
+have built, not just the current one. In our example so far, `Version` is set
+to 10.
+
+With the `ister` tool configured, build the image with with the following command:
+
+.. code-block:: bash
+
+   sudo mixer build image --format 1
+
+This command outputs an image bootable as a virtual machine and which can be
+installed on bare metal.
+
+Mixer automatically looks for the :file:`release-image-config.json` file but
+you can freely choose the filename. To use a different name, simply pass the
+:option:`--template` flag when creating your image, for example:
+
+.. code-block:: bash
+
+   sudo mixer build image --format 1 --template path/to/file.config
+
+By default, `ister` uses the format version of the build machine it runs on.
+Therefore, if the format you are building differs from the format of the |CL|
+OS you are building on, you must use the :option:`--format <FORMAT_NUMBER>`
+flag. Find the current format version of your OS with the following command:
+
+.. code-block:: bash
+
+   sudo cat /usr/share/defaults/swupd/format
+
+Update the next mix version information
+***************************************
+
+Increment the mix version number for the next mix with the following command:
+
+.. code-block:: bash
+
+   sudo mixer versions update
+
+This command automatically updates the mix version stored in the
+:file:`mixversion` file incrementing it by 10. To increment by a different
+amount, use the :option:`--increment` flag, for example:
+
+.. code-block:: bash
+
+   sudo mixer versions update --increment 100
+
+Alternatively, to set the mix version to a specific value, use the
+:option:`--mix-version` flag, for example:
+
+.. code-block:: bash
+
+   sudo mixer versions update --mix-version 200
+
+The :command:`mixer versions update` command does not allow you to set the mix
+version to a value lower than its current value. The mix version is expected
+to always increase, even if the new mix is undoing an earlier change.
+
+If you have been tracking your workspace with Git, you can restore the mix to
+an earlier state but be careful of "rewriting history" if you are publishing
+the mix content to users already.
+
+Update the upstream version of |CL| used as a base for the mix, with the
+following command using the :option:`--upstream-version` flag:
+
+.. code-block:: bash
+
+   sudo mixer versions update --upstream-version 21070
+
+This command also accepts the keyword "latest":
+
+.. code-block:: bash
+
+   sudo mixer versions update --upstream-version latest
+
+This command sets the upstream version to the latest released version of
+upstream |CL| within the same format version. The :command:`mixer
+versions update` command does not allow to set an upstream version to a
```

Comment:
You removed the word "you", but without it, "The `mixer versions update` command does not allow to set" is incorrect.

Reply:
Added 'you' back